### DISCUSSION TITLE
More KH/s

### DISCUSSION THREAD

**Discussion by cryptonitiche** [BitMaker-hub/NerdMiner_v2#432] [Q&A]
Is it possible to do more KH/s? Or 78  KH/s is the maximum?
I think to install a small radiator, and try to raise performance.

**Comment by alcar21** [comment]
It's the maximum. The maximum is 80kh/s but normally the software need to do more thinks like show information in screen, read and send mining information.

**Comment by dwightmulcahy** [comment]
So I recently ran across this: https://github.com/NMminer1024/NMMiner

**_"Deeply optimazation for ESP32-S3, max hashrate: 119 KH/s"_**

![image](https://github.com/user-attachments/assets/7a977bde-20d1-4f15-b3dd-665485a8276d)

**Reply by Lunas2525** [reply]
I am curious as to what speed the miner is running the chip at. From what i can tell nerd miner runs both cores at 40mhz.
By the specs it is supposed to be up to 240mhz going by this yes there should be room to increase the rate in theory up to about 300 mh/s but there is likely a reason why they dont run them with the pedal pinned to the floor. Most likely heat and power. That would destabilize or burn out the chips. With cooling i would think 150- 200mhz should be possible but that is only if im right about what speed the chips are being run at.

**Reply by dwightmulcahy** [reply]
I never see the temp go above 62C and the stats hold at pool.io.  I did add a heat sink to mitigate any overheating.
